botorch/acquisition/objective.py

```diff
@@ -63,7 +63,7 @@ class LinearMCObjective(MCAcquisitionObjective):
     `mc_obj(samples) = sum_{i} weights[i] * samples[..., i]`
 
     Example:
-        >>> # example for a two outcomes
+        Example for a model with two outcomes:
```

Comment:
Sphinx won't correctly parse the following lines as a code block. Here are few options that will be correctly parsed: add a blank line after L66 before the code block, make this line a comment (starting with `>>> #`), or indent the code block to the next level.